### ISSUE TITLE
Lower resolution than Apple TV videos

### ISSUE THREAD

**Issue by chriscalo** [JohnCoates/Aerial#364]
Any reason why these videos aren't at the same resolution as the Apple TV videos? Watching these videos on a 27ʺ iMac next to an Apple TV, the Apply TV videos seem significantly higher in resolution. Are the full-res videos available only to Apple TVs?

**Comment by sn0wyfall** [comment]
Good question. I also noticed that the videos were low quality (they were about 720p to 1080p, as far as I remember). I thought it was strange as Apple most likely has higher resolutions. I'll try to find them.

**Comment by alistairmcmillan** [comment]
@chriscalo @sn0wyfall The videos shown by the Aerial screensaver are the same as shown on the Apple TV. They are listed in this file: http://a1.phobos.apple.com/us/r1000/000/Features/atv/AutumnResources/videos/entries.json

BTW Apple could produce higher resolution videos, but keep in mind (if you check other issues that people have logged here) people are already unhappy about the size of the existing files.

**Comment by chriscalo** [comment]
Have you done a side-by-side comparison of an Apple TV to a 27” iMac? They don't at all appear to be the same resolution. Am I the only one seeing this? Before writing off my report, I would try it and see for yourself if it looks the same. Could there be some other explanation for the poor video quality I'm seeing?

**Comment by alistairmcmillan** [comment]
@chriscalo You're not the first to suggest this. Last time it was suggested the Apple TV was getting videos that the screensaver wasn't I investigated thoroughly.

Set up a shared net connection on my Mac, hooked my Apple TV up to it, factory reset my Apple TV and used Wireshark to inspect all the traffic between the Apple TV and and the internet to watch for the source of the aerial videos and the actual files themselves. Nothing has changed. They are still pulling their videos from the JSON file linked above.

Beyond that, another way we know they are still pulling videos from this JSON file is because when they added new videos in December last year they just added them to the same file. Since they started showing these files with the introduction of the fourth generation Apple TV they haven't changed the source JSON file.

And finally, the Apple TV can only output 1080p. So technically, the videos you are seeing outputted by your Apple TV can't be any higher resolution. https://www.apple.com/apple-tv/specs/

**Comment by sn0wyfall** [comment]
@alistairmcmillan that's a pity! I was really hoping they had some higher resolution versions somewhere. Doesn't make sense with the 1080p Apple TV, though.

**Comment by chriscalo** [comment]
Definitely a shame. Thanks for the detailed explanation. 

Do we think there's some other explanation for the lower image quality? Perhaps how the video is unpacked and rendered on the Mac? The absence of some magic my TV is doing?

**Comment by sn0wyfall** [comment]
@chriscalo Maybe you're looking at it wrong

**Comment by alistairmcmillan** [comment]
@chriscalo I can't answer why there is lower image quality on your iMac compared to your TV. I'd need to know makes and models, and mostly likely see for myself. I can take a guess though.

Do you have a 4K or 5K TV by any chance? If so then, given the dearth of 4K/5K content out there, isn't the TV designed to regular upscale content and make it look good? Won't that be one of its primary selling features? I just did a search for 4K TVs on Amazon, picked the first result and the first listed feature says "...and lift your standard definition content to near 4K quality via LG's six step 4K upscaler"

So, in short, yeah, you're right. Your TV is a magician! :)
